src/Microsoft.Identity.Web/Resource/MicrosoftIdentityIssuerValidatorFactory.cs

```diff
@@ -15,9 +17,20 @@ namespace Microsoft.Identity.Web.Resource
     /// </summary>
     internal class MicrosoftIdentityIssuerValidatorFactory
     {
+        public MicrosoftIdentityIssuerValidatorFactory(
+            IOptions<AadIssuerValidatorOptions> aadIssuerValidatorOptions,
+            IHttpClientFactory httpClientFactory)
+        {
+            _configManager =
+            new ConfigurationManager<IssuerMetadata>(
+                Constants.AzureADIssuerMetadataUrl,
+                new IssuerConfigurationRetriever(),
+                httpClientFactory?.CreateClient(aadIssuerValidatorOptions?.Value?.HttpClientFactoryName));
```

Comment:
What's with all the `?`? None of these things should be null when activated from DI.

Reply:
that's for the unit test to pass, otherwise i have to configure all the service, etc..., which i might do or leave as an integration test instead.